invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities-carousel/index.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2016-2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+import React from "react";
+import ReactDOM from "react-dom";
+import CommunitiesCarousel from "./communities-carousel";
+
+const communitiesCarouselContainer = document.getElementById("communities-carousel");
+const title = communitiesCarouselContainer.dataset.title;
+const fetchUrl = communitiesCarouselContainer.dataset.fetchUrl;
+const intervalDelay = parseInt(communitiesCarouselContainer.dataset.intervalDelay);
+const animationSpeed = parseInt(communitiesCarouselContainer.dataset.animationSpeed);
+const defaultLogo = communitiesCarouselContainer.dataset.defaultLogo;
+
+communitiesCarouselContainer && ReactDOM.render(
```

Comment:
I think we can omit the check now...is better to fail if someone uses that in a page without the id defined rather than not showing something I think...

Reply:
Actually the id-element should always be there, as the js is loaded along with it!